codice/alliance
# CAL-271 added configuring section to templated docs

#### What does this PR do?

Continues updating the managing section by adding the configuring section to the jbake documentation. 
#### Who is reviewing it?

@mcalcote @vinamartin @Lambeaux @jrnorth 

#### Choose 2 committers to review/merge the PR.

@coyotesqrl
@lessarderic
@pklinef
@shaundmorris

#### How should this be tested?

- verify headers added to .adoc files
- review new template (.ftl) files
- check attached pdf against current documentation.pdf to ensure all sections being included.

#### What are the relevant tickets?

[CAL-271](https://codice.atlassian.net/browse/CAL-271)

#### Screenshots (if appropriate)

[jdocumentation.pdf](https://github.com/codice/alliance/files/1046068/jdocumentation.pdf)

#### Checklist:
- [x] Documentation Updated (template)
